brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py

```diff
@@ -0,0 +1,142 @@
+from brainscore_vision.model_helpers.check_submission import check_models
+import functools
+from transformers import AutoFeatureExtractor, CvtForImageClassification
+from brainscore_vision.model_helpers.activations.pytorch import PytorchWrapper
+from PIL import Image
+import numpy as np
+import torch
+
+"""
+Template module for a base model submission to brain-score
+"""
+
+
+def get_model_list(name):
+    assert name == 'cvt_cvt-w24-384-in22k_finetuned-in1k_4'
+    submit_version = 4
+    model_list = ['cvt_cvt-w24-384-in22k_finetuned-in1k_4']
+    return [model + '_' + str(submit_version) for model in model_list]
+
+
+def get_model(name):
+    assert name == 'cvt_cvt-w24-384-in22k_finetuned-in1k_4'
+    # https://huggingface.co/models?sort=downloads&search=cvt
+    image_size = 384
+    processor = AutoFeatureExtractor.from_pretrained('microsoft/cvt-w24-384-22k')
+    model = CvtForImageClassification.from_pretrained('microsoft/cvt-w24-384-22k')
+    preprocessing = functools.partial(load_preprocess_images, processor=processor, image_size=image_size)
+    wrapper = PytorchWrapper(identifier=name, model=model, preprocessing=preprocessing)
+    wrapper.image_size = image_size
+
+    return wrapper
+
+
+def get_layers(name):
+    assert name == 'cvt_cvt-w24-384-in22k_finetuned-in1k_4'
+    # layers = []
```

Comment:
Lines  37-40 can be uncommented out, and line 41 can be deleted. This must have been from an old version of that model that I was forcing it to use a certain layer. We can use all layers now.